src/main/kotlin/mods/eln/transparentnode/LargeRheostat.kt

```diff
@@ -253,7 +253,7 @@ class LargeRheostatGUI(player: EntityPlayer, inventory: IInventory, internal var
     GuiContainerEln(ResistorContainer(player, inventory)) {
 
     override fun postDraw(f: Float, x: Int, y: Int) {
-        helper.drawString(8, 12, -16777216, tr("Nom. Resistance: %1$", Utils.plotValue(render.desc.getRsValue(render.inventory), "Ohm")))
+        helper.drawString(8, 12, -16777216, tr("Nom. Resistance: %1$", Utils.plotValue(render.desc.getRsValue(render.inventory), "Ω")))
```

Comment:
Does the ohm symbol ever need translating?

Reply:
In my language convention, "Ω" has higher recognizability than "Ohm". That's why I changed "Ohm" to "Ω". If you think it's not necessary, I will revert it back.